exercises/poker/src/lib.rs

```diff
@@ -0,0 +1,7 @@
+/// Given a list of poker hands, return a list of those hands which win.
+///
+/// Note the type signature: this function should return _the same_ reference to
+/// the winning hand(s) as were passed in, not reconstructed strings which happen to be equal.
+pub fn winning_hands<'a>(_: &[&'a str]) -> Option<Vec<&'a str>> {
```

Comment:
I would prefer using a variable name instead of `_`

Reply:
1. Any variable name we use will need to begin with an underscore anyway, to prevent compiler warnings about unused variables.
2. Using `_` encourages the implementer choose their own variable name.
3. The only point of including the function stub is so that users don't have to guess and check the expected signature by inspecting the tests. While the name used is technically part of the public signature of the function, I figured it best to minimize the degree to which we constrain the individual implementers.

If you feel very strongly about it, I can change it, but I think there are good reasons to leave it as is.

Reply:
Ok, I'm convinced.